src/main/java/com/tstepnik/planner/domain/BeforeNowValidator.java

```diff
@@ -0,0 +1,26 @@
+package com.tstepnik.planner.domain;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+public class BeforeNowValidator implements ConstraintValidator<NotBeforeNow, ZonedDateTime> {
```

Comment:
Can we stop using this validator and start using future and past? You made one working only with ZonedDateTime the other ones are working with all data/time types. What do you think?

Reply:
Sure, we can change it.